olp-cpp-sdk-core/tests/cache/DefaultCacheTest.cpp

```diff
@@ -219,3 +220,57 @@ TEST(DefaultCacheTest, AlreadyInUsePath) {
   olp::cache::DefaultCache cache2(settings);
   ASSERT_EQ(olp::cache::DefaultCache::OpenDiskPathFailure, cache2.Open());
 }
+
+class ProtectedCacheTest : public ::testing::Test {
```

Comment:
I suggest to move this test fixture and corresponding tests into new file called `ProtectedDefaultCacheTest`